src/main/webapp/app/exercises/programming/assess/programming-assessment-dashboard/programming-exercise-submissions.component.ts

```diff
@@ -112,6 +114,11 @@ export class ProgrammingExerciseSubmissionsComponent implements OnInit {
             .subscribe((submissions: ProgrammingSubmission[]) => {
                 this.submissions = submissions;
                 this.filteredSubmissions = submissions;
+                this.assessedSubmissions = this.submissions.filter((submission) => {
+                    const result = getLatestSubmissionResult(submission);
+                    setLatestSubmissionResult(submission, result);
+                    return result?.rated && Result.isManualResult(result);
```

Comment:
According to the comment above, should `!!result.rated` be used instead of `result?.rated` ?